libgdf/src/io/csv/csv-reader.cu

```diff
@@ -962,18 +967,36 @@ __global__ void convertCsvToGdf(
 
 		if(start>stop)
 			break;
-
+		bool quotation=false;
+		if(raw_csv[pos]=='\"'){
+		  quotation=true;
+		  pos++;
+		}
+ 
 		while(true){
 			if(raw_csv[pos]==delim){
+			  if(quotation==false)
+				break;
+			  else if ((pos-1) >= start && raw_csv[pos-1]=='\"' )
 				break;
 			}
 			else if (raw_csv[pos] == terminator){
+ 			  if(quotation==false)
+				break;
+			  else if ((pos-1) >= start && raw_csv[pos-1]=='\"' )
 				break;
 			}
             else if(raw_csv[pos] == '\r' &&  ((pos+1) < stop && raw_csv[pos+1]=='\n')){
-            	stop--;
+  			  if(quotation==false){
```

Comment:
Indentation.

Reply:
All the indentation issues should be fixed